### PR TITLE
Closehundler

### PR DESCRIPTION
**Pull request by antonovaea**
I made URL with sessionID in /create endpoint, but i can't close session, because it stops in pending status. Need your advise :)

### CODE REVIEW

File: server/store/session-store.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ class SessionStoreInternal extends SessionStore {
             if (session.closeId === closeId) {
                 return session 
             }
+            else if (session.closeId != closeId)
```
Comment (sumbarbarrian) [inline server/store/session-store.js:69]: just use `else`. If `session.closeId` is not equals to `closeId` then `else` block will be called anyway. We don't need additional condition.

File: server/server.js [hunk 2]
```diff
@@ -1,30 +1,35 @@
 const express = require('express')
 const ws = require('ws')
 const cors = require('cors')
+const Session = require("./store/session");
+const SessionStore = require("./store/session-store");
 const server = express()
+let url = new URL("http://localhost:8081/create");
```
Comment (sumbarbarrian) [inline server/server.js:7]: what is it?

File: server/server.js [hunk 3]
```diff
@@ -33,8 +38,25 @@ console.log('create session')
  *  this endpoint closes session
  */
 server.get('/close', (req, res) => {
```
Comment (sumbarbarrian) [inline server/server.js:40]: Might be you forgot to add URL parameter as first.  `server.get('/close/:closeId', ...`
Also use are usign `sessionID` name, are you sure that you use `closeId`, not `sessionId`?

File: server/server.js [hunk 4]
```diff
@@ -1,30 +1,35 @@
...
+
+    let unicCloseToken = uuidv4(); //generating unic token
+    console.log(unicCloseToken)
+
+    let promiseStore = SessionStore.get();
+    promiseStore.then((store) => {
+        store.create(newUuid, unicCloseToken);
+        console.log('create session')
+        let view = { sessionId : newUuid, closeToken: unicCloseToken }
+        url.searchParams.append("sessionID", newUuid);
```
Comment (sumbarbarrian) [inline server/server.js:27]: what do you do here?

File: server/store/session-store.js (hunk 1 above)
Comment (antonovaea) [reply server/store/session-store.js:69]: done

**Pull request closed without merging**